api/app/fireweather_bot/common.py

```diff
@@ -28,7 +30,7 @@ class AuthenticationException(Exception):
     """ Exception thrown if there's any issue authenticating """
 
 
-def _authenticate_session(session: Session) -> Session:
+def authenticate_session(session: Session) -> Session:
```

Comment:
You shouldn't be touching this code - this is for using idir on the old site. The API uses an entirely different mechanism.